practice-two/src/js/script.js

```diff
@@ -1 +1,108 @@
+// Query to id
+const fullNameInput = document.getElementById('fullName');
+const emailInput = document.getElementById('email');
+const salaryInput = document.getElementById('salary');
+const cityInput = document.getElementById('city');
 
+// Check the input condition
+const REGEX = {
+  alphabetRegex: /^[a-zA-Z]+ [a-zA-Z]+$/,
+  emailRegex: /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/
+}
+
+// Fixed value
+const MESSAGE = {
+  empty: 'Empty format',
+  wrongFormat: 'Enter the wrong format. Please re-enter', 
+};
+
+const GENERAL = {
+  correct: ''
+};
+
+function showErrorMessage(input, msg) {
+  const errMessageEl = input.parentElement.querySelector('.warn-msg');
+  errMessageEl.innerHTML = msg;
+}
+
+function isEmpty (input) {
+  if(!input.value) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isValidAlphabet(input) {
+  if (!REGEX.alphabetRegex.test(input.value)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isValidEmail(input) {
+  if (!REGEX.emailRegex.test(input.value)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isValidSalary(input) {
+  if (input.value <= 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+const validateForm = () => {
```

Comment:
function isValidSalary(input) {...}
const validateForm = () => {...}

You are using 2 type of functions. Pls choose one of them